rir/src/interpreter/call_context.h

```diff
@@ -58,7 +58,7 @@ struct CallContext {
     const Code* caller;
     const size_t suppliedArgs;
     size_t passedArgs;
-    const R_bcstack_t* stackArgs;
+    /*const */ R_bcstack_t* stackArgs;
```

Comment:
I don't understand what the commented out const means

Reply:
I removed the 'const' to be able to replace forced promises on the stack with the unwrapped values. Should I remove this for good or add a comment explaining why it is being commented out?

Reply:
I mean if it is not const, then it is not const... I guess I don't understand the reason for commenting out instead of deleting. or what kind of explanation would you write in the coment, other than "it is not const"...

Reply:
got it